.swiftlint.yml

```diff
@@ -53,7 +53,6 @@ opt_in_rules:
   - unneeded_parentheses_in_closure_argument
   - unused_import
   - vertical_whitespace_closing_braces
```

Comment:
I think we can remove this one too. So then we could do:

```

class MyClass {

    var property1
    var property2
    var property3

    func doStuff() { 
       ...     
    }
   
    fund doMoreStuff() { 
        ... 
    }

}
```

Reply:
I say if we remove this, then we also decide that we're going to open every class with an empty space. Don't necessarily need a rule for that though. I'd pref to lose the empty space between the last function and the last brace. But i agree that having a space between the type definition and the first member can help with readability

Reply:
Yeah, I think I'm a fan of getting rid of `vertical_whitespace_opening_braces` but keeping `vertical_whitespace_closing_braces`? I think for me not having a space between the last func and closing brace is a bit easier to read? Basically same as what @jonathajones said. 😄  But also happy to be overruled if anyone feels strongly about it.